src/nl_debug.h

```diff
@@ -0,0 +1,107 @@
+#ifdef KERNEL_MODULE
```

Comment:
Missing license and copyright headers. Let's use `LGPL-2.1-or-later` if `nl_debug` can be used as library in the future?
```
/* SPDX-License-Identifier: LGPL-2.1-or-later */

/*
 * Copyright (C) 2023 Elastio Software Inc.
 */
```

Reply:
Done!